assets/js/autofillTag.js

```diff
@@ -1285,6 +1285,15 @@ const AutofillReplacerTool = (function AutofillReplacerTool() {
         jQuery(headerButtons[z]).prop("disabled", false);
       }
     }
+
+    const contentEditor = cmsIframe.find("div.content-editor-container");
+    const selectOption = cmsIframe.find("div.select-container > div > select");
+
```

Comment:
please include a comment on what this section of code is used for.

Reply:
Done!